lib/mv_opentelemetry.ex

```diff
@@ -32,16 +31,6 @@ defmodule MvOpentelemetry do
   ```
   """
 
-  @doc """
-  Registers an application tracer for your OTP application. Should be called only once
-  per application startup.
-  """
-  @spec register_application(atom()) :: :ok
```

Comment:
I'm thinking maybe to retain it for backwards compatibility, but only stories and insights use it at the moment so who cares, I can remove this code.